src/Examples/Flowsheets/hda_ideal_VLE.py

```diff
@@ -0,0 +1,1025 @@
+##############################################################################
```

Comment:
We can use the file in the Tutorials section. I put a duplicate here hoping that it would be easier to access the file. Please let me know if you prefer using the one in the Tutorials section, I will remove this file.

Reply:
Duplicating files is never a good idea - always use an existing file if there is one.